floodwatch/src/js/components/CustomFilter.js

```diff
@@ -122,3 +140,98 @@ export class CustomFilter extends Component {
   }
 
 }
+
+
+
+type LocationStateType = {
+  value: string,
+  items: Array<Object>,
+  highlightedStyle: {
```

Comment:
Good question--think they were just leftovers of a previous way of styling the dropdown.